packages/suite/src/support/messages.ts

```diff
@@ -3072,9 +3093,21 @@ export default defineMessages({
         id: 'TR_YOU_WERE_DISCONNECTED_DOT',
     },
     TR_YOUR_CURRENT_FIRMWARE: {
-        defaultMessage: 'Your current firmware version is {version}',
+        defaultMessage: 'Your current version is {version}',
```

Comment:
You are deleting word `firmware` here but in firmware type default message it is there